CONTRIBUTING.md

```diff
@@ -63,26 +66,26 @@ to pass it on as an open-source contribution.
 > By making a contribution to this project, I certify that:
```

Comment:
The indent here is coming from the upstream text, I'm not too keen on having us alter it.